R/gs_cp.R

```diff
@@ -16,27 +16,32 @@
 #  You should have received a copy of the GNU General Public License
 #  along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-
-#' @title Group sequential design - Conditional Probability under non-proportional hazards
-#' @description  \code{gs_cp()} computes conditional boundary crossing probabilities at future planned analyses
-#' for a given group sequential design assuming an interim z-statistic at a specified interim analysis.
-#' @param x An object of type \code{gsDesign2}
-#' @param i analysis at which interim z-value is given; must be from 1 to max(x$analysis$analysis)
-#' @param zi interim z-value at analysis i (scalar)
-#' @return A list with: theta -- a list containing theta0 (theta under H0, i.e., 0,) and theta1 (a vector of theta under H1)
-#'                      upper_bound -- the upper bound value return by any gs_design_ahr function
-#'                      upper_prob -- a list contains the conditional probability given zi under H0 and H1, respectively
+#' Conditional probability under non-proportional hazards for
+#' group sequential design
 #'
-#' @examples
-#' library(gsDesign2)
-#' library(dplyr)
+#' Computes conditional boundary crossing probabilities at future planned
+#' analyses for a given group sequential design assuming an interim
+#' z-statistic at a specified interim analysis.
+#'
+#' @param x An object of type `gs_design`.
```

Comment:
Thanks, @nanxstats! One minor comments. The `x` is usually an output from `gs_update_ahr`.

As the `gs_cp` is still in progress, there are many upcoming changes to the documentation. If it's acceptable to you, could we complete the development first and then refine the documentation?

Reply:
Okay! I'll close this for now.